test/gridengine/setup_tool.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cp tool_conf.xml config
+cp galaxy.ini /etc/galaxy/galaxy.ini
```

Comment:
Is this galaxy.ini file really needed? What is changed in contrast to the original one? Can we change these setting with GALAXY_CONFIG_* settings?